### PR TITLE
[CNXC-255] Reformat overall layout

### PR DESCRIPTION
**Pull request by janakitti**
### Problem Statement
The overall UI layout for COVID-Net is inconsistent.

### Implementation
- Added grey padding around the Create Analysis page components
- Removed the Dashboard header
- Set a minimum height for the past analyses table and patient DICOM search results so the contain extends to the bottom of the viewport
- Ensured consistent padding sizes

### CODE REVIEW

File: src/assets/scss/components/CreateAnalysisDetail.scss [hunk 1]
```diff
@@ -24,7 +23,7 @@ $description-top-padding: 0.3rem;
 .detail-top-wrapper {
   @include flex(row);
   width: $full;
-  padding: 10px;
+  margin: $page-section-margin;
```
Comment (boris-nguyen) [inline src/assets/scss/components/CreateAnalysisDetail.scss:26]:
```suggestion
  margin: $page-section-margin;
  
```

File: src/assets/scss/components/PatientLookup.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .InputRow {
   display: flex;
   flex-direction: row;
-  margin: 1rem;
+  margin: $page-section-margin;
```
Comment (boris-nguyen) [inline src/assets/scss/components/PatientLookup.scss:4]:
```suggestion
  margin: $page-section-margin;
  
```

File: src/components/CreateAnalysis/CreateAnalysis.tsx [hunk 3]
```diff
@@ -47,15 +46,11 @@ const CreateAnalysis: React.FC = () => {
...
-    <Stack>
-      <StackItem>Create a new Predictive analysis</StackItem>
-      <StackItem isFilled>
-        <PatientLookup isOnDashboard={true}></PatientLookup>
-        {/* <Button onClick={submitFile}>Choose File...</Button> */}
-      </StackItem>
-    </Stack>
+    <>
+      <PatientLookup isOnDashboard={true}></PatientLookup>
+      {/* <Button onClick={submitFile}>Choose File...</Button> */}
```
Comment (boris-nguyen) [inline src/components/CreateAnalysis/CreateAnalysis.tsx:51]: Shall we remove this unused code? If yes, also remove `<>` at the same time. @janakitti

File: src/components/PatientLookup.tsx [hunk 4]
```diff
@@ -1,4 +1,6 @@
-import { Button,TextInput } from "@patternfly/react-core";
+import {
+  Button, Stack, StackItem, TextInput
+} from '@patternfly/react-core';
```
Comment (boris-nguyen) [inline src/components/PatientLookup.tsx:3]:
```suggestion
} from "@patternfly/react-core";
```

File: src/assets/scss/components/dashboard.scss [hunk 5]
```diff
@@ -38,22 +36,21 @@ $sectionAreaSpacing: 15px;
...
-  position: absolute;
-  padding-left: 1rem;
   padding-top: 5px;
 }
 
-.PastAnalysis{
-  position: relative;
+.PastAnalysis {
+  flex-grow: 1;
+  margin: $page-section-margin;
```
Comment (boris-nguyen) [inline src/assets/scss/components/dashboard.scss:44]:
```suggestion
  margin: $page-section-margin;
  
```

File: src/components/PatientLookup.tsx [hunk 6]
```diff
@@ -68,7 +70,9 @@ const PatientLookup: React.FC<PatientLookupProps> = ({ isOnDashboard }) => {
 
   return (
     <React.Fragment>
-      <div className="InputRow">
+      <Stack>
+        <StackItem className="input-row-label">Create a new Predictive analysis</StackItem>
```
Comment (boris-nguyen) [inline src/components/PatientLookup.tsx:74]: Should this be "Create a new predictive analysis" or "Create a New Predictive Analysis"?

The capitalization seems a bit inconsistent.

@colinywu @janakitti what are your thoughts on this?

File: src/components/pastAnalysis/PastAnalysisTable.tsx [hunk 7]
```diff
@@ -291,32 +291,36 @@ const PastAnalysisTable = () => {
...
+              <InputGroupText> <SearchIcon /> </InputGroupText>
+            </InputGroup>
+          </div>
+
+          <div style={{ float: "right" }}>
+            <button className="pf-c-button pf-m-inline pf-m-tertiary pf-m-display-sm" type="button" style={{ marginRight: "1em" }} onClick={() => updatePage(-1)} disabled={isLoading || tableStates.page == 0}>
+              <span className="pf-c-button__icon pf-m-end">
+                <i className="fas fa-arrow-left" aria-hidden="true"></i>
+              </span>
+          &nbsp; Previous {perpage}
```
Comment (boris-nguyen) [inline src/components/pastAnalysis/PastAnalysisTable.tsx:313]:
```suggestion
            &nbsp; Previous {perpage}
```

File: src/components/pastAnalysis/PastAnalysisTable.tsx [hunk 8]
```diff
@@ -291,32 +291,36 @@ const PastAnalysisTable = () => {
...
+            </button>
+            <button className="pf-c-button pf-m-inline pf-m-tertiary pf-m-display-sm" type="button" onClick={() => updatePage(1)} disabled={isLoading || tableStates.page === tableStates.lastPage}>Next {perpage}
```
Comment (boris-nguyen) [inline src/components/pastAnalysis/PastAnalysisTable.tsx:315]:
```suggestion
            <button className="pf-c-button pf-m-inline pf-m-tertiary pf-m-display-sm" type="button" onClick={() => updatePage(1)} disabled={isLoading || tableStates.page === tableStates.lastPage}>
              Next {perpage}
```

**Review by boris-nguyen — APPROVED** [review]
Great work @janakitti

Loving the new layout, it looks very clean and consistent.

The only super minor comment that I left is about capitalization of the "Create a new predictive analysis" title. Otherwise, it's good to go.

File: src/components/PatientLookup.tsx (hunk 6 above)
Comment (boris-nguyen) [reply src/components/PatientLookup.tsx:74]: Resolving this comment, has been fixed in another PR.

**Comment by boris-nguyen** [comment]
Resolved the comment about capitalization.
[PR#38](https://github.com/darwinai/covidnet_ui/pull/38/files) corrects the capitalization.

**Review by colinywu — APPROVED** [review]
Tested and confirmed it works, and looks great! The responsiveness is also much improved with your new changes, and overall look is more consistent. Great work with the new styling properties

**Pull request merged**